docs/content/concepts/dynamic-fields/transfers/transfer-to-object.mdx

```diff
@@ -72,11 +76,23 @@ module sui::transfer {
     /// Given mutable (i.e., locked) access to the `parent` and a `Receiving`
     /// object referencing an object owned by `parent` use the `Receiving` ticket
     /// and return the corresponding object.
+    ///
+    /// This function has custom rules enforced by the Sui Move bytecode verifier that ensures
```

Comment:
```suggestion
    /// This function has custom rules enforced by the Sui Move bytecode verifier that ensure
```

Reply:
Also, are these doc comments also going to be (or have been) applied to the actual framework code? It would be good to.